src/main/java/mods/railcraft/common/plugins/ic2/crops/CropCreosote.java

```diff
@@ -0,0 +1,59 @@
+//package mods.railcraft.common.plugins.ic2.crops;
+
+//import ic2.api.crops.CropCard;
+//import ic2.api.crops.CropProperties;
+//import ic2.api.crops.ICropTile;
+//import mods.railcraft.common.items.RailcraftItems;
+//import net.minecraft.item.ItemStack;
+//import net.minecraft.init.Items;
+
+/**
+ * Created by GeneralCamo on 4/27/2017.
+ *
+ * @author GeneralCamo
+ *         Created for Railcraft <http://www.railcraft.info>
+ */
+//public class CropCreosote extends RailcraftCropCard {
+//
+//    public String getId() {
+//        return "creosote";
+//    }
+//
+//    public String getDiscoveredBy() {
+//        return "Forecaster";
```

Comment:
I plan on putting my own name for the Nickel crop I plan on doing. That one should be relatively easy, just need a bit of art since I can just extend the base metal crop from IC2.